packages/preview-middleware/src/base/flex.ts

```diff
@@ -76,9 +77,17 @@ export async function readChanges(project: ReaderCollection, logger: Logger): Pr
  * @returns object with success flag and optional message
  */
 export function writeChange(
-    data: object & { fileName?: string; fileType?: string },
+    data: object & {
+        fileName: string;
```

Comment:
`fileName` and `fileType` were optional properties before and we're checking if they exist on line 94. We probably should not change the type definitions for those properties.